packages/tokens/config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     'src/tokens/color/component-light.json',
     'src/tokens/color/semantic-dark.json',
     'src/tokens/color/component-dark.json',
+    'src/tokens/spacing/spacing.json',
   ],
   platforms: {
```

Comment:
Should there also be a `figma` export so it also leverages the tokens package as a source of truth?

Reply:
Do we know what the level of effort would be to create a Figma export? Originally, I didn’t suggest creating one since there are a lot fewer spacing tokens than color tokens. It’d be pretty easy for me to manually maintain them and will probably only take me an hour to add them to Figma. But if it’s pretty easy to create the Figma export, it definitely couldn’t hurt to have it.